File: customer-address-jpa/src/main/java/com/izertis/example/service/dtos/CustomerSearchCriteria.java
package com.izertis.example.service.dtos;

import java.io.Serializable;

/** CustomerSearchCriteria. */
@lombok.Getter
@lombok.Setter
public class CustomerSearchCriteria implements Serializable {

    private String name;

    private String email;

    private String city;

    private String state;

}

File: kustomer-address-jpa/src/main/kotlin/com/izertis/example/service/dtos/CustomerSearchCriteria.kt
package com.izertis.example.service.dtos

import java.io.Serializable

/**
 * CustomerSearchCriteria.
 */
data class CustomerSearchCriteria(


    val name: String?  = null,


    val email: String?  = null,


    val city: String?  = null,


    val state: String?  = null

)  : Serializable {






}
